src/components/Learn-Components/Card-Wrapper/index.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
```

Comment:
This is not a component, rather this code should be under the sections. We have a folder under that `Learn-Layer5`, please use that.